### DISCUSSION TITLE
MySQL

### DISCUSSION THREAD

**Discussion by JenniDiaz1102** [Launch-X-Latam/MisionFrontEnd#542] [Team Posts]
Hola compañeros me podrian ayudar con este problema por favor se los agradeceria mucho. Estoy usando windows.

![MySQL Workbench 09_11_2022 09_14_18 p  m](https://user-images.githubusercontent.com/114269494/200996024-6c057715-619e-41f5-853d-25f353402c4c.png)
![MySQL Workbench 09_11_2022 09_04_07 p  m](https://user-images.githubusercontent.com/114269494/200996180-52af0718-8fc6-41d6-b651-37a1573ab952.png)

**Comment by nilovili** [comment]
Creo que no tienes mysql instalado en tu equipo, por eso no encuentra ningun servidor, porque MySQL Workbench solo es un entorno gráfico, podrías instalar docker y una imagen de mysql e integrarlo con el mysql workbench. como este video. https://www.youtube.com/watch?v=2zlP6n7kQ68&t=681s 


> Hola compañeros me podrian ayudar con este problema por favor se los agradeceria mucho. Estoy usando windows.

**Comment by CarloMorale** [comment]
Primero que nada ¿Te aseguraste de tener instalado el MySQL Server?

**Comment by NelsonOrtizLeal** [comment]
Si, lo que dice @CarloMorale deberias ver. Si quieres asegurarte de haberlo instalando. Una forma facil es desde MySQL Installer. Deberia aparecer Workbench y Server.

![imagen](https://user-images.githubusercontent.com/114643735/201006847-ab0983d7-4df9-457c-8747-765b36245b85.png)

Si fuese el caso que no lo tienes, le das en el boton Add y agregar MySQL Server
